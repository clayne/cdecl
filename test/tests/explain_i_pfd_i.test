cdecl @ @ @ explain int x, (*f)(double) @ 0
