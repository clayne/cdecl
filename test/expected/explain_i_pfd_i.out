declare x as integer
declare f as pointer to function (double precision) returning integer
